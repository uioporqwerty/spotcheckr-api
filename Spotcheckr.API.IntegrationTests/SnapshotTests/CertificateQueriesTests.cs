using HotChocolate.Execution;
using Snapshooter.Xunit;
using Xunit;

namespace Spotcheckr.API.IntegrationTests.SnapshotTests
{
	public class CertificateQueriesTests : BaseSnapshotTest
	{
		[Fact]
		public async void GetCertificates_WithGet_ReturnsCertificates()
		{
			var executor = await GetRequestExecutorAsync();
			var result = await executor.ExecuteAsync(@"
				query {
					certificates() {
						id
						code
						description
					}
				}
			");

			result.MatchSnapshot();
		}
	}
}
