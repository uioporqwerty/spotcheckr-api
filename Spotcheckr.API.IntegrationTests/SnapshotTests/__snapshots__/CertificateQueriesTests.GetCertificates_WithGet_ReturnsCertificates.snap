{
  "data": {
    "certificates": [
      {
        "code": "NASM-CPT",
        "description": "Some cert description"
      },
      {
        "code": "NASM-CPP",
        "description": "Some other cert description"
      }
    ]
  }
}
